util/walk/walk.go

```diff
@@ -436,6 +439,12 @@ func Depth(p string) uint {
 }
 
 func createPathScanner(path string) *bufio.Scanner {
+	escape := map[rune]bool{
```

Comment:
For a tiny memory optimization, can you change this to a `map[rune]struct{}` please?